packages/webapp/src/components/navbar/Navbar.tsx

```diff
@@ -4,14 +4,22 @@ import Toolbar from '@material-ui/core/Toolbar';
 import Typography from '@material-ui/core/Typography';
 import Avatar from "@material-ui/core/Avatar";
 import PersonIcon from '@material-ui/icons/Person';
+import { useHistory } from 'react-router'
+import HomeLogo from './Logo.png'
```

Comment:
this file can be kept outside, at the location of board.svg. 
Or 
We can create a static files folder called asset, and keep these things there.